src/js/electron/userTasks.ts

```diff
@@ -18,7 +18,7 @@ export default function(app: any) {
         iconPath: process.execPath,
         iconIndex: 0,
         title: "Move Brim to Current Display",
-        description: "Move brim windows to the current display."
+        description: "Move Brim windows to the current display"
```

Comment:
shoot sorry, I also should have mentioned the inconsistency w "Current Display". Not trying to be capitalization police, they're just right next to each other so I can't not see it haha 🙈

Reply:
I think the title is like title cased like a book cover, but the description is a normal sentence. So the lower cased words work there.

Reply:
Oh gotcha, yes that works!